user-guide/Reference/DataMiner_Extension_Modules.md

```diff
@@ -20,6 +20,12 @@ For ease of installation, DxMs may be packaged together in a pack, such as the [
 
 The APIGateway module is the endpoint for the gRPC communication with a DataMiner Agent.
 
+From DataMiner 10.4.2/10.5.0 onwards<!--RN 37734 + 36575-->, DataMiner modules can register with APIGateway. All modules registered with APIGateway are displayed under `/APIGateway/api/version`, showing the following properties:
```

Comment:
It is yet to be merged but it is likely it will also become available on 10.4.0MR together with 10.4.2FR